<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rating</class>
 <widget class="QWidget" name="Rating">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
border-radius:10px;
border: 4px solid rgb(163, 137, 131);
background-color: rgb(255, 240,245);
}
QLabel{
font: 75 16pt &quot;Times New Roman&quot;;
border: 1px solid rgb(163, 137, 131);
}
QLabel#label_3,QLabel#label_5,QLabel#label_7,QLabel#label_9,QLabel#label_11,QLabel#label_13,QLabel#label_15,QLabel#label_16,QLabel#label_20{
background-color: rgb(236, 215, 210);
}
QLabel#label,QLabel#label_18,QLabel#label_19{
border: 0px solid rgb(163, 137, 131);
font: 75 18pt &quot;Times New Roman&quot;;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>571</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>51</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>91</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ошибки:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>551</width>
      <height>561</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>311</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string>при приведении к каноническому
 виду:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="maximumSize">
         <size>
          <width>311</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string>при вычислении θ:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="maximumSize">
         <size>
          <width>311</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string>при выборе базиса/min θ:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="maximumSize">
         <size>
          <width>311</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string>при привидении к базису:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="maximumSize">
         <size>
          <width>311</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string>при вычисление ∆k:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="maximumSize">
         <size>
          <width>311</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string>при оценивании ∆k:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>311</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string>Итого:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="maximumSize">
         <size>
          <width>311</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string>Число обращений к комплексу:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>141</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>141</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>141</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>141</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>141</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>141</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>141</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>141</width>
          <height>51</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>40</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Вариант:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
